plugins/catalog-react/src/filters.ts

```diff
@@ -159,3 +163,28 @@ export class UserListFilter implements EntityFilter {
     return this.value;
   }
 }
+
+/**
+ * Filters entities based if it is an orphan or not.
+ * @public
+ */
+export class EntityOrphanFilter implements EntityFilter {
+  constructor(readonly values: string[]) {}
+  filterEntity(entity: Entity): boolean {
+    const orphan = entity.metadata.annotations?.['backstage.io/orphan'];
+    return orphan !== undefined && this.values.includes(orphan);
+  }
+}
+
+/**
+ * Filters entities based on if it has errors or not.
+ * @public
+ */
+export class EntityErrorFilter implements EntityFilter {
+  constructor(readonly values: string[]) {}
+  filterEntity(entity: Entity): boolean {
+    const error =
+      ((entity as AlphaEntity)?.status?.items?.length as number) > 0;
+    return error !== undefined && this.values.includes(error.toString());
```

Comment:
It seems a bit odd to be comparing string representations of booleans. Would it make sense to use a boolean member variable instead? eg. `constructor(readonly value: boolean)`

Reply:
Yes, totally agree, I've made this change 👍